apps/admin-portal/src/components/identity-providers/wizard/steps/wizard-summary.tsx

```diff
@@ -117,6 +119,17 @@ export const WizardSummary: FunctionComponent<WizardSummaryProps> = (
             {
                 authenticatorSummary?.properties && getAuthenticatorProperties()
             }
+        </>;
+    };
+
+    const isAuthenticatorStepAvailable = () => {
+        return identityProvider?.federatedAuthenticators?.defaultAuthenticatorId;
+    };
+
+    return (
+        <Grid className="wizard-summary">
+            {getGeneralDetailsComponent()}
```

Comment:
Let's keep a space before and after curly brackets. Please check other places as well.

Reply:
Fixed in 6311ed7b87d8256e38389b5145f845d46e4c8564.